build.gradle

```diff
@@ -154,6 +158,21 @@ configurations.all {
     System.setProperty("org.gradle.internal.http.socketTimeout", 120000 as String)
 }
 
+task updateGTNHBuildScript {
+    doLast {
+        if (checkForUpdate()) {
+            def f = new File("build.gradle")
+        new URL("https://raw.githubusercontent.com/SinTh0r4s/ExampleMod1.7.10/main/build.gradle")
+                .withInputStream { i -> f.withOutputStream { it << i } }
```

Comment:
I prefer to never use single letter names, what do you think about changing them to full names? Like `builscriptFile` and `newVersionInputStream` ?

Reply:
Leftover from testing. Whoops